reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -23,9 +23,34 @@ class IterativeRefinement:
     particles : arr
         Particles to be reconstructed.
         Shape (n_particles, n_pix, n_pix)
-    ctf_info : list of dicts
-        Each dict contains CTF k,v pairs per particle.
-            Shape (n_particles,)
+    ctf_info : dict
```

Comment:
I wouldn't repeat. Just say, see *** documentation and point to the place the ctf comes from. that way when it changes there it doesn't need to be updated here.

Reply:
also the documentation is too restrictive. while these are reasonable/typical values for testing, others outside their range are still valid, even if a bit less common.